### ISSUE TITLE
'Learn More' link in Services section

### ISSUE THREAD

**Issue by cunning-folk** [CunningFolkDev/elevate360_website#84]
I noted that the 'Learn More' links are not right aligned in the services section. The other sections seem sorted

**Comment by cunning-folk** [comment]
<img width="473" alt="screen shot 2018-11-28 at 5 34 06 pm" src="https://user-images.githubusercontent.com/31627480/49191480-f25f1780-f333-11e8-8d64-efcc21a7f811.png">

**Comment by gblakeman** [comment]
Yes, I left #68 open, intending to make a separate pass on these links tomorrow or Friday. Getting those links in place is oddly-complicated in some situations and I didn’t want to let it slow me down as I worked on other top-level bits.
